scripts/table_updates/update_table_schema.py

```diff
@@ -19,185 +16,356 @@ def copy_table_schema(syn, from_table_id, to_table_id):
     to_table_schema.columnIds = from_table_schema.columnIds
     return to_table_schema
 
+
 def create_synapse_column(name, col_type, max_size):
     """
-    Create a new Synapse Column with given info
+    Create a new Synapse Column with given info. ??? may extend, string. use data element column types
     """
-    if col_type in ['INTEGER','DOUBLE','LARGETEXT']:
-        new_column = Column(name=name,
-                            columnType=col_type)
+    if col_type in ["INTEGER", "DOUBLE", "LARGETEXT"]:
+        new_column = Column(name=name, columnType=col_type)
     else:
         max_size = 500 if pandas.isna(max_size) else int(max_size)
-        new_column = Column(name=name,
-                            columnType=col_type, 
-                            maximumSize=max_size)
+        new_column = Column(name=name, columnType=col_type, maximumSize=max_size)
     return new_column
 
+
 def _expand_checkbox_vars(row):
+    """Expand checkbox variables into multiple columns
+
+    Args:
+        row (pd.Series): A row of the data element dataframe
+
+    Returns:
+        pandas.DataFrame: A dataframe with expanded checkbox variables
+    """
     temp_df_list = []
-    if not pandas.isna(row['colLabels']):
-        for label in row['colLabels'].split(','):
+    if not pandas.isna(row["colLabels"]):
+        for label in row["colLabels"].split(","):
             temp_df = {}
-            temp_df['col_name'] = row['variable']+"___"+label
-            temp_df['synColType'] = row['synColType']
-            temp_df['synColSize'] = row['synColSize']
-            temp_df['variable'] = row['variable']
+            temp_df["col_name"] = row["variable"] + "___" + label
+            temp_df["synColType"] = row["synColType"]
+            temp_df["synColSize"] = row["synColSize"]
+            temp_df["variable"] = row["variable"]
             temp_df_list.append(temp_df)
         return pandas.DataFrame(temp_df_list)
 
+
+def _get_latest_table(form) -> str:
+    """get the latest table for each form
+
+    Args:
+        form (tuple): the tuple of the form and the corresponding tables outputed by groupby
```

Comment:
nit: could you rephrase and give more details on what is in this tuple, e.g: 

```
form(Tuple[str, Dict, str]):  the tuple containing:
        - str: form
        - dict: Mapping of gene symbols to metadata
        - str: some string
```
Not sure what you meant by groupby, what groupby are you referring to here?

Reply:
example of groupby: `form_groups = master_table_view.groupby("form")`. It gives a tuple of form name and a dataframe.